### PR TITLE
Feature/ersc/rdphoen 1143 dmverity boot

### PR DESCRIPTION
**Pull request by iris-ersc**
Introduce dm-verity for the two rootfs partitions

- change uuu script
- change init script in initramfs
- fix blocksize for ext4 image creation

### CODE REVIEW

File: recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu [hunk 1]
```diff
@@ -76,6 +76,18 @@ FBK: acmd zcat | dd of=/dev/mapper/irma6lvm-rootfs_b
 FBK: ucp rootfs.ext4.gz t:-
 FBK: ucmd sync
 
+# Create fat data partition (keystore)
+FBK: ucmd mkfs.vfat /dev/mapper/irma6lvm-keystore
+FBK: ucmd mkdir -p /mnt/keystore
+FBK: ucmd mount -t vfat /dev/mapper/irma6lvm-keystore /mnt/keystore
+
+# TODO: Unsafe! Place roothash in a secure manner!
```
Comment (Jasper-Ben) [inline recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu:84]: Can we have a Jira task here, so that we don't forget to fix this? :)

**Review by Jasper-Ben — APPROVED** [review]

**Review by linkjumper — APPROVED** [review]

File: recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu (hunk 1 above)
Comment (iris-ersc) [reply recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu]: RDPHOEN-1163

File: recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu [hunk 2]
```diff
@@ -76,6 +76,19 @@ FBK: acmd zcat | dd of=/dev/mapper/irma6lvm-rootfs_b
 FBK: ucp rootfs.ext4.gz t:-
 FBK: ucmd sync
 
+# Create fat data partition (keystore)
+FBK: ucmd mkfs.vfat /dev/mapper/irma6lvm-keystore
+FBK: ucmd mkdir -p /mnt/keystore
+FBK: ucmd mount -t vfat /dev/mapper/irma6lvm-keystore /mnt/keystore
+
+# !! The root hashes for the dm-verity volumes are generated here !!
+# TODO: Unsafe! Store roothash in a secure manner!
```
Comment (DanielBaumgart) [inline recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu:85]: Is there a jira task for this TODO?

**Review by DanielBaumgart — APPROVED** [review]

File: recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu (hunk 2 above)
Comment (iris-ersc) [reply recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu:85]: RDPHOEN-1163

Comment (DanielBaumgart) [reply recipes-bsp/irma6-uuu/files/flashall_imx8mp.uuu:85]: Then i would prefer to also mention this on the comment in the code.

**Pull request merged**